### DISCUSSION TITLE
Support with Sidechain: Sidechains with quorum

### DISCUSSION THREAD

**Discussion by archm49** [ConsenSys/quorum#1359] [Q&A] [answered]
Does the latest version of quorum work with sidechain(blockchain)?

**Comment by antonydenyer** [comment, marked as answer]
yes GoQuorum can work with sidechains, it depends on what you're doing. The main factor at the moment is that there is no Gas in GoQuorum so you wouldn't want to make your network public.

https://ethereum.org/en/developers/docs/scaling/sidechains/
